src/sharedFramework/sharedFramework.proj

```diff
@@ -106,6 +105,19 @@
       <runtimeGraphGeneratorRuntime Condition="'$(runtimeGraphGeneratorRuntime)'==''">linux</runtimeGraphGeneratorRuntime>      
     </PropertyGroup>
 
+    <ItemGroup>
+      <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnetapphost" />
+      <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnethostresolver" />
+      <TrimPkgsFromDeps Include="microsoft.netcore.dotnetapphost" />
+      <TrimPkgsFromDeps Include="microsoft.netcore.dotnethostresolver" />
+      <TrimPkgsWithCmdPrefix Include="@(TrimPkgsFromDeps ->'--remove %(Identity)')" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <TrimPkgsCmd>@(TrimPkgsWithCmdPrefix)</TrimPkgsCmd>
+      <TrimPkgsCmd>$(TrimPkgsCmd.Replace(';', ' '))</TrimPkgsCmd>
```

Comment:
You don't need to do it this way.  Instead you can do something like the following:

`<RestoreSourceArg>@(RestoreTestSource->'--source %(Identity)', ' ')</RestoreSourceArg>`

which puts the space as a separator from the start.